### ISSUE TITLE
Keys file

### ISSUE THREAD

**Issue by monsty** [cryptonotefoundation/cryptonote#166]
Hi,

Is there any way to generate a .keys file ?
I saw in the SimpleWallet.cpp that we can use a "file.keys" to load the wallet.

Can I make a transfer with only the wallet address and a key ?

**Comment by rainmanp7** [comment]
If you use the simple wallet you are generating the key
To transfer you ,would need to carry the contents on the wallet with the key.
You can't run around with the keys to the car and have no car right?
Or could you use the key and hit reset to the blockchain?

Are you assuming the key is your paperwallet as an example?

I think your asking if you can recreate your wallet with just the key it's self?

Please be more simplified as in give an example?

Transfers are generally made with a wallet address and a key. So for
transferring funds ,I would think ,this is how it's normally done. I might
Be right or wrong ,but you could try it :-)

Thank you.

**Comment by SoraKohaku** [comment]
@phfirecoin is key for sha public or what?he not explain some about that
key. there key public and private key i think....address is in simplewallet

On Feb 2, 2018 8:55 AM, "PHFirecoin" <notifications@github.com> wrote:

If you you the simple wallet you are generating the key
To transfer you ,would need to carry the contents on the wallet with the
key.
You can't run around with the keys to the car and have no car right?
Or could you use the key and hit reset to the blockchain?

I think your asking if you can recreate your wallet with just the key it's
self?

Please be more simplified as in give an example please?

Thank you.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/cryptonotefoundation/cryptonote/issues/166#issuecomment-362441326>,
or mute the thread

**Comment by monsty** [comment]
Yes, I want to be able to recreate any wallet with just the key it self so I can save it on a paper anywhere.

I want to be able to transfer money with only the informations (address / key) I printed on the paper, because all the other informations is on the block chain right ?

Maybe I’m wrong, how does MyEtherWallet work ? Because when I create a wallet I only need to save the private keys and not the whole wallet file. Are they storing the wallet file for us ?

**Comment by SoraKohaku** [comment]
building cryptonote token still on progress around 70%

**Comment by SoraKohaku** [comment]
ah ok i on way made it. i put on develop tree. i will input your suggest on
my document thank you~

**Comment by monsty** [comment]
Ok nice ! I can help you to build this feature :)

**Comment by SoraKohaku** [comment]
tommorow will added you on. welcome aboard to family~

**Comment by SoraKohaku** [comment]
Happy Explore bro [Here]([url]https://github.com/FndNur1Labs/DirhamCli/tree/notetoken(url)) @monsty

**Comment by SoraKohaku** [comment]
https://github.com/FndNur1Labs/DirhamCli/tree/notetoken

**Comment by Goldeneum** [comment]
yes, me too, i need this feature... it make no sense to open the wallet by file. the correct way is to open it by public+private keys.
is there any one can help with this, please?

**Comment by SoraKohaku** [comment]
little time after im not busy continue version CryptoNote with codename
Ruby finish. i just have busy real life. i already solve this puzzle later
soon i put on my repo or maybe some PR but that not work. ahaha. patient
guys

On Feb 7, 2018 8:28 PM, "Goldeneum" <notifications@github.com> wrote:

yes, me too, i need this feature... it make no sense to open the wallet by
file. the correct way is to open it by public+private keys.
is there any one can help with this, please?

—
You are receiving this because you commented.

Reply to this email directly, view it on GitHub
<https://github.com/cryptonotefoundation/cryptonote/issues/166#issuecomment-363740756>,
or mute the thread
